exercises.md

```diff
@@ -30,7 +30,7 @@ List extracted from problem-specifications repo on 2022-06-03
 | [counter][counter] | . |
 | [crypto-square][crypto-square] | . |
 | [custom-set][custom-set] | . |
-| [darts][darts] | . |
+| [darts][darts] | [implemented](https://github.com/exercism/awk/pull/24) |
```

Comment:
In hindsight, is this file helpful?

Reply:
Eh. Only as a way to track what exercises are needed, which can be done via Github or the configlet. So ... not super helpful.